samples/isolates/src/sample/isolates.cljd

```diff
@@ -0,0 +1,103 @@
+(ns sample.isolates
+  "An example following https://blog.codemagic.io/understanding-flutter-isolates/"
+  (:require ["package:flutter/material.dart" :as m]
+            ["dart:isolate" :as di]
+            ["dart:core" :as dc]
+            [cljd.flutter :as f]))
+
+(defn heavy-computation-task
+  [my-send-port]
+  (let [mike-receive-port (di/ReceivePort.)]
+    (->> mike-receive-port
+         .-sendPort
+         (.send ^di/SendPort my-send-port))
+    (-> mike-receive-port
+        (.forEach
+          (fn [[my-message
+                coffee-type
+                mike-response-send-port]]
+            (.send ^di/SendPort
+                   mike-response-send-port
+                   (str [my-message
+                         coffee-type])))))))
+
+(defn isolate-trigger!
+  [receive-port
+   response-receive-port
+   {::keys [message-box
+            type]}]
+  (let [mike-send-port (await (.-first ^di/ReceivePort receive-port))]
+    (.send ^di/SendPort
+           mike-send-port
+           [message-box
+            type
+            (.-sendPort ^di/ReceivePort response-receive-port)])))
+
+
+(defn isolate-test-ui
+  []
+  (f/widget
+    :let [mike-response-receive-port (di/ReceivePort.)]
```

Comment:
mike or make ^^?

Reply:
no need for the `.` for constructors in cljd

Reply:
I’ve changed to my, mike was in the initial example but it diverged a little bit in the sample that I’ve made